real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/messages/BrakeStateMessage.java

```diff
@@ -15,33 +15,38 @@
 public class BrakeStateMessage extends BaseMessage {
 
 	public static final String VERSION_ID = "BRAKESTATEV0.1";
-	private int state;
+	private boolean isDown;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Field isDown has the same name as a method](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2343306721/issues/source?bid=2938665&fileBranchId=3212596#l18)